resources/lang/en/events/lead_ceremony_sc.json

```
@@ -139,7 +139,7 @@
       "rank": ["warrior", "deputy"],
       "life_giving": [
         {
-          "text": "r_c walks up to m_c, offering a life for [virtue]. {PRONOUN/r_c/subject/CAP} {VERB/r_c/smile/smiles}, and {VERB/r_c/state/states} that the Clan will do well under m_c's leadership.",
+          "text": "r_c walks up to m_c, offering a life for [virtue]. {PRONOUN/r_c/subject/CAP} {VERB/r_c/smile/smiles} and {VERB/r_c/say/says} that the Clan will do well under m_c's leadership.",
```

```suggestion
          "text": "r_c walks up to m_c, offering a life for [virtue]. {PRONOUN/r_c/subject/CAP} {VERB/r_c/smile/smiles} and {VERB/r_c/state/states} that the Clan will do well under m_c's leadership.",
```
No need to change this imo, "they state XYZ" is more powerful than "they say XYZ". it's its own "trust me bro"

---

ah yeah fair enough... I kinda just beef with "states" as a verb because I feel like 99.9% of the time it's far too stiff and awkward to justify itself. Maybe this can be the 0.1% exception.